### DISCUSSION TITLE
Question About Speech to Text Using C++

### DISCUSSION THREAD

**Discussion by dominickairmyne** [deepgram/community#654] [General help] [answered]
Hi! Has anyone gotten basic Speech to text working with C++? I am trying to implement it on ESP32 but I am kind of lost! Thanks in advance!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by dominickairmyne** [reply]
I have none of these just a basic question!

**Comment by SandraRodgers** [comment, marked as answer]
I believe you got your answer in Discord but just want to link this resource in case others come across this question:

https://github.com/deepgram-devs/code-samples (Check out the example in C++)
